src/Places/Suggestions/Suggestions.ts

```diff
@@ -0,0 +1,34 @@
+import { Client } from 'Client'
```

Comment:
Is the `from 'Client'` intentional?

Reply:
Yes, it is! This is the type we use on the constructor!